extensions/cms/package.json

```diff
@@ -518,7 +518,7 @@
 			{
 				"command": "cms.resource.deleteRegisteredServer",
 				"title": "%cms.resource.deleteRegisteredServer.title%",
-				"when": "viewItem == cms.resource.itemType.registeredServer"
+				"when": "viewItem == cms.resource.itemType.databaseServer"
```

Comment:
where the name `databaseServer` come from?  this is a bit confusing to me since we already work with "databases" and "servers".

Reply:
This is because right now the way a node looks for context actions is by looking at "database" and "servers" in a regex in the itemType.

Reply:
Could you explain this some more? Why was registeredServer causing a problem?

---
In reply to: [289212851](https://github.com/microsoft/azuredatastudio/pull/5542#discussion_r289212851) [](ancestors = 289212851)